### PR TITLE
Fix misc. grammar and whitespace

### PR DESCRIPTION
**Pull request by luzpaz**
Grammar and whitespace fixes

---

- [x]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8866">pipeline status</a> for feature branch [PR_8866](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8866). Pipeline [804423042](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/804423042) was triggered at [b60f473](https://github.com/FreeCAD/FreeCAD/pull/8866/commits/b60f4737bbc638b287bac13e95c215d86dab8375). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Base/Factory.h [hunk 1]
```diff
@@ -49,8 +49,8 @@ class BaseExport AbstractProducer
 
 
 /** Base class of all factories
-  * This class has the purpose to produce at runtime instances
-  * of classes not known at compile time. It holds a map of so called
+  * This class has the purpose to produce a runtime instance
+  * of classes unknown at compile time. It holds a map of so called
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply src/Base/Factory.h:53]: Apologies,@wwmayer. Does it need a revert or can it be improved ?

[the remark this answers is not in the document]
Comment (wwmayer) [reply src/Base/Factory.h:53]: Maybe:
This class has the purpose to produce instances of classes at runtime that are unknown at compile time.

[the remark this answers is not in the document]
Comment (luzpaz) [reply src/Base/Factory.h:53]: Done in #8882

**Pull request merged**